From d33ce73c1ff6ca6bd6b7dc0aae02461399d08e24 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 27 Nov 2015 11:10:53 +0800
Subject: [PATCH 1117/1691] MLK-11915-03 ASoC: imx-sii902x: init sii902x_np to
 avoid wild pointer

commit d33ce73c1ff6ca6bd6b7dc0aae02461399d08e24 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

init sii902x_np to avoid wild pointer. Reported by Coverity.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
(cherry picked from commit 780e27683c38f785ea7f7e07e83b00ffd3c22d78)
---
 sound/soc/fsl/imx-sii902x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/imx-sii902x.c b/sound/soc/fsl/imx-sii902x.c
index 72c407d..7cd19fa 100644
--- a/sound/soc/fsl/imx-sii902x.c
+++ b/sound/soc/fsl/imx-sii902x.c
@@ -172,7 +172,7 @@ static struct snd_soc_ops imx_sii902x_ops = {
 
 static int imx_sii902x_probe(struct platform_device *pdev)
 {
-	struct device_node *cpu_np, *sii902x_np;
+	struct device_node *cpu_np, *sii902x_np = NULL;
 	struct platform_device *cpu_pdev;
 	struct imx_sii902x_data *data;
 	int ret;
-- 
1.9.1

